### PR TITLE
Feature/issue 59

### LINKED ISSUE #59: Create Swipe component

### PR DESCRIPTION
**Pull request by BCommeine**
close #59 
#284

### CODE REVIEW

**Comment by BCommeine** [comment]
Did something wrong while solving a merge conflict, just let me take care of it

**Comment by DemianD** [comment]
@BertCommeine If you need help, ping me

File: web-app/src/components/Polaroid/Polaroid.scss [hunk 1]
```diff
@@ -16,4 +17,9 @@
...
 .Polaroid img {
     width: 100%;
-}
\ No newline at end of file
+    user-select: none;
+    -moz-user-select: none;
+    -webkit-user-drag: none;
+    -webkit-user-select: none;
+    -ms-user-select: none;
+}
```
Comment (DemianD) [inline web-app/src/components/Polaroid/Polaroid.scss]: There are all fallback for browser, but there's one that doesn't fit in this list: ```-webkit-user-drag: none;```
Also, I guess that the fallbacks are auto generated by sass (correct me if 'm wrong)

File: web-app/src/components/Polaroid/Polaroid.scss [hunk 2]
```diff
@@ -16,4 +17,9 @@
 
 .Polaroid img {
     width: 100%;
-}
\ No newline at end of file
+    user-select: none;
+    -moz-user-select: none;
+    -webkit-user-drag: none;
```
Comment (DemianD) [inline web-app/src/components/Polaroid/Polaroid.scss]: There are all fallback for browser, but there's one that doesn't fit in this list: ```-webkit-user-drag: none;```
Also, I guess that the fallbacks are auto generated by sass (correct me if 'm wrong)

**Review by DemianD — CHANGES_REQUESTED** [review]
Except of that one nitpick, the code looks good! 👍

File: web-app/src/components/Observations/Observations.js [hunk 3]
```diff
@@ -45,11 +45,16 @@ class Observations extends Component {
               <div className="col col-lg-12">
                 <img className="Observations__PolaroidIcon" src={polaroid} alt="Polaroid camera" />
               </div>
-              <div className="col col-lg-offset-2 col-lg-8">
-                <div className="Observations__Picture">
-                  <Polaroid img={generateImageUrl(observation.id)} />
+              <Swipeable
+                onSwipeLeft={() => vote(1)}
```
Comment (DemianD) [inline web-app/src/components/Observations/Observations.js]: Left is -1

**Review by DemianD — CHANGES_REQUESTED** [review]

File: web-app/package.json [hunk 4]
```diff
@@ -10,12 +10,16 @@
     "lodash": "^4.17.4",
     "react": "^15.6.1",
     "react-dom": "^15.6.1",
+    "react-draggable": "^2.2.6",
```
Comment (DemianD) [inline web-app/package.json]: We are not using this

File: web-app/package.json [hunk 5]
```diff
@@ -10,12 +10,16 @@
...
     "react-pinch-zoom-pan": "^1.4.2",
     "react-redux": "^5.0.5",
     "react-router-dom": "^4.1.1",
+    "react-swing": "^0.0.7",
+    "react-touch": "^0.4.2",
```
Comment (DemianD) [inline web-app/package.json]: We are not using this

**Review by DemianD — CHANGES_REQUESTED** [review]

**Pull request merged**